app/src/main/java/com/example/workout_timer/MainActivity.java

```diff
@@ -55,20 +68,92 @@ public boolean onOptionsItemSelected(MenuItem item) {
     }
 
     public void decrementRounds(View view) {
+        rounds--;
+        displayRounds();
     }
 
     public void incrementRounds(View view) {
+        rounds++;
+        displayRounds();
     }
 
     public void decrementWork(View view) {
+        workTime -= 1000;
+        displayWorkTimerQuantity();
     }
 
     public void incrementWork(View view) {
+        workTime += 1000;
+        displayWorkTimerQuantity();
     }
 
     public void decrementRest(View view) {
+        restTime -= 1000;
+        displayRestTimerQuantity();
     }
 
     public void incrementRest(View view) {
+        restTime += 1000;
+        displayRestTimerQuantity();
+    }
+
+    private void setUpFAB() {
+        ExtendedFloatingActionButton fab = findViewById(R.id.startButton);
+        fab.setOnClickListener(view -> {
```

Comment:
This part needs work. I am not even sure why I have WorkTimer and RestTimer in MainActivity at all. I'm thinking they are only actually necessary in a ActivityTimer.